tutorial.md

```diff
@@ -1,101 +1,85 @@
 
-The first thing to do is to select the Android device you want to capture on. If
-your device does not shows up in the "Device" list, you can refresh the device
-list by clicking on the reload arrow next to the drop-down menu.
+    <img src="../images/system_profiler/capture-dialog.png" width="500px">
 
-<img src="../images/system_profiler/capture-dialog.png" width="500px">
+5. In "Type", select "System profile".
 
-In "Type", select "System profile".
+6. Select which application you want to trace. 
 
-Next, you need to select which application you want to trace. AGI comes with a
-simple Vulkan application that we are going to use as an example here. Click on
-the "..." button right next to the "Application" text entry to list applications
-that can be traces on the selected device. This pops-up the "Select an
-Application to Trace" window. Type "gapid" in the filter text box to filter down
-applications to the ones that contain "gapid" in their package name. Expand the
-package, select "com.google.android.gapid.VkSampleActivity", and press "OK". This
-brings you back to the capture options window, with the "Application" field
-populated by the Application you selected.
+    <div class="info" markdown="span">	
+    AGI comes with a simple Vulkan application that we are going to use as an example.<br><br>
+    1. Click on the "..." button right next to the "Application" text entry to list applications that can be traced on the selected device.<br>
+    2. This pops-up the "Select an Application to Trace" window.<br>
+    3. Type "gapid" in the filter text box to filter down applications to the ones that contain "gapid" in their package name.<br>
+    4. Expand the package, select "com.google.android.gapid.VkSampleActivity", and press "OK".<br>
+    5. This brings you back to the capture options window, with the "Application" field populated by the Application you selected.<br>
+    </div>
 
-You may also choose another Vulkan application if you prefer.  If there is only
-one activity in the package, you can simply select the package instead of the Activity.
+    You may also choose another Vulkan application if you prefer.  If there is only one activity in the package, you can simply select the package instead of the Activity.
 
-In the "Application" section, you can leave the other fields empty.
+7. In the "Application" section, you can leave the other fields empty.
 
-Under "Start and Duration" section, set "Start at" to "Manual", and "Duration"
-to 2.
+8. Under "Start and Duration" section, set "Start at" to "Manual", and "Duration" to 2.
 
-In the "Trace Option" section, click on the "Configure" button to pop-up a
-configuration window where you can select which can of performance counters you
-want to be captured.
+9. In the "Trace Option" section, click on the "Configure" button to pop-up a configuration window where you can select which can of performance counters you want to be captured.
```

Comment:
... where you can select which ~~can of~~ performance counters ..

Reply:
See https://github.com/google/agi/pull/182/commits/62bf1b179b61c5e8ae25464df3961aa218ff1bd8